src/main/java/at/researchstudio/sat/merkmalservice/model/Organization.java

```diff
@@ -0,0 +1,36 @@
+package at.researchstudio.sat.merkmalservice.model;
```

Comment:
Also, this PR will break the build once its merged, since the pom.xml version is not bumped and thus github actions will not be able to push the asset onto the repository

Reply:
done